repos/spack_repo/builtin/packages/toml_f/package.py

```diff
@@ -2,30 +2,46 @@
 #
 # SPDX-License-Identifier: (Apache-2.0 OR MIT)
 
+from spack_repo.builtin.build_systems import cmake, meson
+from spack_repo.builtin.build_systems.cmake import CMakePackage
 from spack_repo.builtin.build_systems.meson import MesonPackage
 
 from spack.package import *
 
 
-class TomlF(MesonPackage):
-    """
-    TOML parser implementation for data serialization and deserialization in Fortran
-    """
+class TomlF(MesonPackage, CMakePackage):
+    """TOML parser implementation for data serialization and deserialization in Fortran"""
 
-    homepage = "https://toml-f.readthedocs.io"
-    url = "https://github.com/toml-f/toml-f/releases/download/v0.3.1/toml-f-0.3.1.tar.xz"
-    git = "https://github.com/toml-f/toml-f.git"
+    homepage = "https://toml-f.readthedocs.io/"
+    url = "https://github.com/toml-f/toml-f/releases/download/v0.4.2/toml-f-0.4.2.tar.xz"
+    git = "https://github.com/toml-f/toml-f/"
 
-    maintainers("awvwgk")
+    maintainers("mtaillefumier")
```

Comment:
```suggestion
    maintainers("awvwgk", "mtaillefumier")
```
? Unless @awvwgk wants to be removed...